src/coord/rmr/redise.c

```diff
@@ -93,15 +114,17 @@ MRClusterTopology *RedisEnterprise_ParseTopology(RedisModuleCtx *ctx, RedisModul
         return NULL;
       }
     } else if (AC_AdvanceIfMatch(&ac, "RANGES")) {  // End of general arguments
-      int rc = AC_GetU32(&ac, &numShards, AC_F_GE1);
+      int rc = AC_GetU32(&ac, &numRanges, AC_F_GE1);
```

Comment:
So `RANGES` MUST be last?
```
CLUSTERSET
  RANGES <total-number-of-distinct-from-to-ranges>
  MYID <my_id>
```
would fail here, as far as I understand. Is that the behavior that we agreed upon?